src/landing/Validations.js

```diff
@@ -0,0 +1,36 @@
+export const validateUsername = (username) => {
+  if (username.length === 0) {
+    return false;
+  }
+  return true;
+};
+
+export const validatePassword = (password) => {
+  if (password.length < 6) {
+    return false;
+  }
+  return true;
+};
+
+export const validateFirstName = (firstName) => {
+  // eslint-disable-next-line max-len
+  if (firstName.length === 0 || firstName.charAt(0) !== firstName.charAt(0).toUpperCase()) {
+    return false;
+  }
+  return true;
+};
+
+export const validateLastName = (lastName) => {
+  // eslint-disable-next-line max-len
```

Comment:
Please try to handle those without escaping the rule.

Reply:
fixed